### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Portfolio: Desktop Version

In the portfolio i:

- Added media query for a desktop version

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Emmanuel Ogah, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has failing linter checks on PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

File: assets/css/styles.css [hunk 1]
```diff
@@ -1616,3 +1629,1033 @@ form button:active {
   background: #0d151e;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 1200px) {
```
Comment (SajjadAhmad14) [inline assets/css/styles.css:1633]: Please use value for min-width property should in range of 992px - 1100px.

File: index.html [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
           <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Logo"></a>
         </div>
+        <ul class="nav-item">
+          <li class="nav-item1"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-item2"><a href="#about">About</a></li>
+          <li class="nav-item3"><a href="#contact">Contact</a></li>
+        </ul>
         <div class="menu-icon">
           <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Menue"></a>
         </div>
```
Comment (SajjadAhmad14) [inline index.html:28]: Please follow the design for the navigation bar. Kindly see the screenshot below and make changes accordingly.
nav

File: index.html [hunk 3]
```diff
@@ -29,7 +34,7 @@
     <section class="header-page">
```
Comment (SajjadAhmad14) [inline index.html:34]: The text of the headline section and social icons should be at the center of the page. Moreover, please add more space between social icons. KIndly see the screenshot below
headline
.

File: index.html [hunk 4]
```diff
@@ -29,7 +34,7 @@
     <section class="header-page">  
       <div class="header-container">
           <div class="headline">
-            <h1>I’m Emmanuel Glad to see <br/> you!</h1>
+            <h1>I’m Emmanuel Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             <div class="social-media">
```
Comment (SajjadAhmad14) [inline index.html:40]: Please use the `ul` tag as a container for social icons and every icon should be wrapped inside a `li` tag.

File: index.html [hunk 5]
```diff
@@ -80,7 +85,7 @@ <h3>Tonic</h3>
         </div>
       </div>
       <div class="card">
-        <img src="assets/images/mobile/portfolio3.svg" alt="project1">
+        <div class="project3"></div>
         <div class="card-text">
           <div class="card-header">
             <h3>Tonic</h3>
```
Comment (SajjadAhmad14) [inline index.html:91]: Projects do not follow the design on laptop view. KIndly see the screenshot below and fix the issues.
project1

Comment (SajjadAhmad14) [inline index.html:91]: The `See Project` button is missing on the large laptop screen for some projects. Kindly see the screenshot below.
missing2

File: index.html [hunk 6]
```diff
@@ -172,7 +177,7 @@ <h3>Multi-Post
         </div>
       </div>
     </section>
-    <section class="about" id="about">
+  <section class="about" id="about">
```
Comment (SajjadAhmad14) [inline index.html:180]: The About section does not follow the design on laptop view. Kindly see the screenshot below.
about

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Emmanuel Ogah, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (Emmy-github-webdev) [reply index.html:28]: This screenshot seems to be for mobile device,s not desktop

image


I need clarification. Thank you

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Emmy-github-webdev,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mke2111) [inline index.html:28]: - [ ] As previously requested, for desktop version, the navbar should change and not look like the mobile version navbar.
Screenshot from 2021-10-02 10-21-56

File: index.html [hunk 7]
```diff
@@ -168,11 +172,12 @@ <h3>Multi-Post
...
                 </div>
               </div>
+            </div>
           </div>
         </div>
       </div>
-    </section>
-    <section class="about" id="about">
+    </section> 
+  <section class="about" id="about">
```
Comment (mke2111) [inline index.html:180]: - [ ] The about section is still missing a few details to match the original design.

File: assets/css/styles.css [hunk 8]
```diff
@@ -321,13 +329,33 @@ header {
...
+  background-repeat: no-repeat;
+}
+
+.project4 {
+  background-image: url("../images/mobile/portfolio4.svg");
+  background-repeat: no-repeat;
+}
+
 .card-text {
   display: flex;
```
Comment (GzimAsani) [inline assets/css/styles.css:353]: - [ Optional ] You have used this command 93 times, this falls under bad practicing CSS. I would highly recommend you to create a class with this command and use it whenever you need in your HTML.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         

## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**